### YamlMime:ManagedReference
items:
- uid: System.Printing.PrintingCanceledException
  id: PrintingCanceledException
  children:
  - System.Printing.PrintingCanceledException.#ctor
  - System.Printing.PrintingCanceledException.#ctor(System.String)
  - System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String)
  - System.Printing.PrintingCanceledException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Printing.PrintingCanceledException.#ctor(System.String,System.Exception)
  - System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.Exception)
  - System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.String,System.String,System.Int32)
  - System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.String,System.String,System.Int32,System.Exception)
  langs:
  - csharp
  name: PrintingCanceledException
  nameWithType: PrintingCanceledException
  fullName: System.Printing.PrintingCanceledException
  type: Class
  summary: "Undantag som uppstår när koden försöker få åtkomst till ett avbrutna utskriftsjobb."
  remarks: "Om du skapar en [!INCLUDE[TLA#tla_winclient](~/add/includes/ajax-current-ext-md.md)] program som innehåller funktioner för utskrift och tillåter utskriftsjobb avbrytas, bör du hanterar det här undantaget. Ett ohanterat undantag av den här typen kan orsaka ett program för Windows Presentation Foundation slutar svara.      > [!CAUTION] > Klasserna i den <xref:System.Printing>namnområde stöds inte för användning i en Windows-tjänst eller ASP.NET-program eller tjänst.</xref:System.Printing> Försök att använda dessa klasser från inom någon av följande programtyper kan ge oväntade problem, så som minskat tjänsten prestanda- och undantag.       Om du vill skriva ut från en Windows Forms-program finns i <xref:System.Drawing.Printing>namnområde.</xref:System.Drawing.Printing>"
  syntax:
    content: 'public class PrintingCanceledException : System.Printing.PrintJobException'
  inheritance:
  - System.Exception
  - System.SystemException
  - System.Printing.PrintSystemException
  - System.Printing.PrintJobException
  implements: []
  inheritedMembers:
  - System.Printing.PrintJobException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Printing.PrintJobException.JobId
  - System.Printing.PrintJobException.JobName
  - System.Printing.PrintJobException.PrintQueueName
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor
  id: '#ctor'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException()
  nameWithType: PrintingCanceledException.PrintingCanceledException()
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException()
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initierar en ny instans av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klass."
  syntax:
    content: public PrintingCanceledException ();
    parameters: []
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException(String)
  nameWithType: PrintingCanceledException.PrintingCanceledException(String)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(String)
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initierar en ny instans av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klass som innehåller ett specifikt fel."
  syntax:
    content: public PrintingCanceledException (string message);
    parameters:
    - id: message
      type: System.String
      description: "En <xref:System.String>som beskriver feltillståndet.</xref:System.String>"
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String)
  id: '#ctor(System.Int32,System.String)'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException(Int32,String)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String)
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initierar en ny instans av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klass som innehåller en felkod och feltillstånd."
  syntax:
    content: public PrintingCanceledException (int errorCode, string message);
    parameters:
    - id: errorCode
      type: System.Int32
      description: "En <xref:System.Int32>som tolkas som HRESULT en kodad numeriskt värde som har tilldelats ett specifikt undantag.</xref:System.Int32>"
    - id: message
      type: System.String
      description: "En <xref:System.String>som beskriver feltillståndet.</xref:System.String>"
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException(SerializationInfo,StreamingContext)
  nameWithType: PrintingCanceledException.PrintingCanceledException(SerializationInfo,StreamingContext)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initierar en ny instans av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klass som innehåller specifika <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> och <xref:System.Runtime.Serialization.StreamingContext>.</xref:System.Runtime.Serialization.StreamingContext> Den här konstruktorn är skyddad."
  syntax:
    content: protected PrintingCanceledException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "De data som krävs för att serialisera eller deserialisering av objekt."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Kontexten, inklusive källa och mål för den serialiserade dataströmmen."
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException(String,Exception)
  nameWithType: PrintingCanceledException.PrintingCanceledException(String,Exception)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(String,Exception)
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initierar en ny instans av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klass som innehåller ett specifikt fel villkor, inklusive den underliggande orsaken till undantaget."
  syntax:
    content: public PrintingCanceledException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "En <xref:System.String>som beskriver feltillståndet.</xref:System.String>"
    - id: innerException
      type: System.Exception
      description: "Det underliggande felet villkor som orsakade det <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref>."
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.Exception)
  id: '#ctor(System.Int32,System.String,System.Exception)'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException(Int32,String,Exception)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String,Exception)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String,Exception)
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initierar en ny instans av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klass som innehåller en felkod och feltillstånd, inklusive den underliggande orsaken till undantaget."
  syntax:
    content: public PrintingCanceledException (int errorCode, string message, Exception innerException);
    parameters:
    - id: errorCode
      type: System.Int32
      description: "En <xref:System.Int32>som tolkas som HRESULT en kodad numeriskt värde som har tilldelats ett specifikt undantag.</xref:System.Int32>"
    - id: message
      type: System.String
      description: "En <xref:System.String>som beskriver feltillståndet.</xref:System.String>"
    - id: innerException
      type: System.Exception
      description: "Det underliggande felet villkor som orsakade det <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref>."
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.String,System.String,System.Int32)
  id: '#ctor(System.Int32,System.String,System.String,System.String,System.Int32)'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException(Int32,String,String,String,Int32)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32)
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initierar en ny instans av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klass som innehåller en felkod, feltillstånd, utskriftskön namn, Jobbnamn och jobb-ID."
  syntax:
    content: public PrintingCanceledException (int errorCode, string message, string printQueueName, string jobName, int jobId);
    parameters:
    - id: errorCode
      type: System.Int32
      description: "En <xref:System.Int32>som tolkas som HRESULT en kodad numeriskt värde som har tilldelats ett specifikt undantag.</xref:System.Int32>"
    - id: message
      type: System.String
      description: "En <xref:System.String>som beskriver feltillståndet.</xref:System.String>"
    - id: printQueueName
      type: System.String
      description: "Namnet på utskriftskön värd för utskriftsjobbet som orsakade undantaget."
    - id: jobName
      type: System.String
      description: "Namnet på utskriftsjobbet som orsakade undantaget."
    - id: jobId
      type: System.Int32
      description: "Utskriftsjobb som orsakade undantaget ID-nummer."
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.String,System.String,System.Int32,System.Exception)
  id: '#ctor(System.Int32,System.String,System.String,System.String,System.Int32,System.Exception)'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException(Int32,String,String,String,Int32,Exception)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32,Exception)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32,Exception)
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initierar en ny instans av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klass med felkod, fel, utskriftskön namn, jobbnamn, jobb-ID och underliggande orsaken till undantaget."
  syntax:
    content: public PrintingCanceledException (int errorCode, string message, string printQueueName, string jobName, int jobId, Exception innerException);
    parameters:
    - id: errorCode
      type: System.Int32
      description: "En <xref:System.Int32>som tolkas som HRESULT en kodad numeriskt värde som har tilldelats ett specifikt undantag.</xref:System.Int32>"
    - id: message
      type: System.String
      description: "En <xref:System.String>som beskriver feltillståndet.</xref:System.String>"
    - id: printQueueName
      type: System.String
      description: "Namnet på utskriftskön värd för utskriftsjobbet som orsakade undantaget."
    - id: jobName
      type: System.String
      description: "Namnet på utskriftsjobbet som orsakade undantaget."
    - id: jobId
      type: System.Int32
      description: "Utskriftsjobb som orsakade undantaget ID-nummer."
    - id: innerException
      type: System.Exception
      description: "Den bakomliggande orsaken till undantaget."
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Printing.PrintJobException
  isExternal: false
  name: System.Printing.PrintJobException
- uid: System.Printing.PrintingCanceledException.#ctor
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException()
  nameWithType: PrintingCanceledException.PrintingCanceledException()
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException()
- uid: System.Printing.PrintingCanceledException.#ctor(System.String)
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException(String)
  nameWithType: PrintingCanceledException.PrintingCanceledException(String)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String)
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException(Int32,String)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String)
- uid: System.Int32
  parent: System
  isExternal: true
  name: Int32
  nameWithType: Int32
  fullName: System.Int32
- uid: System.Printing.PrintingCanceledException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException(SerializationInfo,StreamingContext)
  nameWithType: PrintingCanceledException.PrintingCanceledException(SerializationInfo,StreamingContext)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Printing.PrintingCanceledException.#ctor(System.String,System.Exception)
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException(String,Exception)
  nameWithType: PrintingCanceledException.PrintingCanceledException(String,Exception)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.Exception)
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException(Int32,String,Exception)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String,Exception)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String,Exception)
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.String,System.String,System.Int32)
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException(Int32,String,String,String,Int32)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32)
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.String,System.String,System.Int32,System.Exception)
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException(Int32,String,String,String,Int32,Exception)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32,Exception)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32,Exception)
- uid: System.Printing.PrintingCanceledException.#ctor*
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException
  nameWithType: PrintingCanceledException.PrintingCanceledException
